server/lib/template/partials/enterpriseDetails.handlebars

```diff
@@ -1,12 +1,13 @@
 {{#if metadata.enterprise.logo}}
   <div class="col-xs-1">
-    <img src="{{absolutePath}}/{{slashed metadata.enterprise.logopath}}" width="65" height="65" alt="">
+    {{!-- <img src="{{absolutePath}}/{{slashed metadata.enterprise.logopath}}" width="65" height="65" alt=""> --}}
+    <img src="data:image/jpg;base64,{{metadata.enterprise.binaryLogo}}" width="65" height="65" alt="">
```

Comment:
One thing to consider ... this is always `image/jpg`.  Should we limit all uploads to be `jpg` or should we try to convert pngs -> jpgs, or is there a way to figure out the image type ...?

Reply:
It seems like pngs are also handled by `image/jpg`

Reply:
:dancer: